### PR TITLE
Updates to add.data.sh

### PR DESCRIPTION
**Pull request by serbinsh**
@robkooper   Can you check this, but I believe these changes are in order

### CODE REVIEW

File: scripts/add.data.sh [hunk 1]
```diff
@@ -21,6 +21,7 @@ addInputFile "${FQDN}" "297" "FAO_" "${FAO_INPUT}"
 addInputFile "${FQDN}" "296" "OGE2_" "${OGE2_INPUT}"
 
 # [76] EBIFARM
+addInput "76" "24" "2004-01-01" "2009-12-31"
```
Comment (robkooper) [inline scripts/add.data.sh]: Where is this input used? If you want to reference it use ${INPUT_ID} which is the id of the newly added input

**Comment by mdietze** [comment]
@serbinsh just checking in to see if more commits are coming to respond to @robkooper or whether the pull request should be canceled

**Comment by serbinsh** [comment]
@mdietze lets cancel this for now....I will make a new commit later after discussing with @robkooper

**Comment by serbinsh** [comment]
@mdietze or @robkooper How do I "cancel" this pull request?

**Comment by serbinsh** [comment]
Do I just "close" this? Does that cancel it?

**Comment by serbinsh** [comment]
This is no longer valid.

**Pull request closed without merging**